src/react-compat.js

```diff
@@ -107,6 +113,26 @@ if (REACT013) {
     throw e;
   }
 
+  // Shallow renderer is accessible via the react-test-renderer package for React 15.5+.
+  // This is a separate package though and may not be installed.
+  try {
+    if (REACT155) {
+      // eslint-disable-next-line import/no-extraneous-dependencies
+      shallowRendererFactory = require('react-test-renderer/shallow').createRenderer;
+    } else {
+      // eslint-disable-next-line import/no-extraneous-dependencies
+      shallowRendererFactory = TestUtils.createRenderer;
+    }
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error(
+      'react-test-renderer is an implicit dependency in order to support react@15.5+. ' +
```

Comment:
@ljharb I'd rather not change this in this PR seeing as the existing error messages also use multiline strings